src/app/netzgrafik-application/netzgrafik-application.component.ts

```diff
@@ -48,13 +52,24 @@ export class NetzgrafikApplicationComponent {
       .subscribe((params) => {
         uiInteractionService.setEditorMode(EditorMode.NetzgrafikEditing);
         uiInteractionService.showNetzgrafik();
-        versionControlService.load(params.getVariantId(), true);
+        try {
```

Comment:
Is there a reason why this is a `try`/`catch` instead of a `if (disableBackend)`?